packages/react-scripts/template-typescript/public/index.html

```diff
@@ -22,9 +25,7 @@
     <title>React App</title>
   </head>
   <body>
-    <noscript>
-      You need to enable JavaScript to run this app.
-    </noscript>
+    <noscript> You need to enable JavaScript to run this app. </noscript>
```

Comment:
I could have sworn I commented on this earlier. This all looks fine but the spacing inside this tag is weird. It's not a big deal, just kind of weird choice on behalf of prettier.

Reply:
Oh wow, that is weird. I wonder if it's a bug.

Reply:
I think it's _technically_ correct because the whitespace that was there when things were on multiple lines does get turned into a single space as that's just how whitespace in HTML works. But preserving it in this case seems like an odd choice.

Reply:
Yeah, we can probably safely remove it.

Reply:
Definitely. I think Prettier is preserving it because in some cases you might be relying on that space. Think about how whitespace works in JSX in a multiline block. So I think Prettier is just playing it safe and leaving it there for you. But we definitely don't need it in this case.

Reply:
I've removed the extra spaces.

Reply:
There is an open issue in prettier for handling whitespace efficiently https://github.com/prettier/prettier/issues/5439